tests/integration/conftest.py

```diff
@@ -721,22 +721,9 @@ def default_activation_instance_logs(
             "type": "string",
             "secret": True,
         },
-        {
-            "id": "ssh_key_data",
-            "label": "SCM Private Key",
-            "type": "string",
-            "format": "ssh_private_key",
-            "secret": True,
-            "multiline": True,
-        },
-        {
-            "id": "ssh_key_unlock",
-            "label": "Private Key Passphrase",
-            "type": "string",
-            "secret": True,
-        },
     ]
 }
+INJECTORS = {"extra_vars": {"username": "Fred"}}
```

Comment:
`{"extra_vars": {"password": "{{ password }}"}}` to better simulate the real use

Reply:
The RBAC autogenerated test data don't have a defined `password`, which will cause tests to fail.

Reply:
updated